main.nf

```diff
@@ -3353,7 +3315,62 @@ def svdb_merge_panel_version(task) {
 	cat <<-END_VERSIONS > ${task.process}_versions.yml
 	${task.process}:
 	    svdb: \$( echo \$(svdb) | head -1 | sed 's/usage: SVDB-\\([0-9]\\.[0-9]\\.[0-9]\\).*/\\1/' )
-	    samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
+	END_VERSIONS
+	"""
+}
+
+process postprocess_merged_panel_sv_vcf {
+
+	cpus 2
+	cache 'deep'
```

Comment:
TODO: remove this

Reply:
Why is this needed?

Reply:
Leftover from svdb_merge_panel process which formed the skeleton for this process. I'm guessing it's not needed.